io/genbank/genbank.go

```diff
@@ -406,12 +514,20 @@ type parseLoopParameters struct {
 // method to init loop parameters
 func (params *parseLoopParameters) init() {
 	params.newLocation = true
-	params.feature.Attributes = make(map[string]string)
+	params.feature.Attributes = multimap.NewMapSlice[string, string]()
 	params.parseStep = "metadata"
 	params.genbankStarted = false
 	params.genbank.Meta.Other = make(map[string]string)
 }
 
+func parseError(line string, lineNum int, err error, info string) error {
+	if err == nil {
+		return fmt.Errorf("Line %d malformed (%s): %s", lineNum, info, line)
+	} else {
+		return fmt.Errorf("Line %d malformed (%s): %s\n%v", lineNum, info, line, err)
+	}
+}
```

Comment:
Add documentation to this. Why was it added, etc.

Reply:
replaced with new error struct, documentation added

Reply:
I like it.